Dockerfile

```diff
@@ -1,3 +1,43 @@
+##
+# Build Stage
+#
+FROM ubuntu:focal as build
+
+ENV DEFAULT_PW="adminADMIN!"
+
+##
+# Copy Source Code
+#
+ADD ./pom.xml /build/pom.xml
+ADD ./tools /build/tools
+ADD ./src /build/src
+ADD ./install /build/install
+ADD ./dev /build/dev
+
+WORKDIR /build
+
+##
+# Checkout Dependencies
+#
+RUN git clone https://github.com/I-TECH-UW/dataexport.git --branch develop
+RUN git clone https://github.com/openelisglobal/openelisglobal-plugins.git plugins --branch master
+
+##
+# Build DataExport
+#
+WORKDIR /build/dataexport
+RUN mvn clean install -DskipTests
+
+WORKDIR /build
+
+# OE Default Password
+RUN ${DEFAULT_PW} | ./install/createDefaultPassword.sh
```

Comment:
@pmanko 
ive tried this locally but it doesnt seem to be able to pass the [password value ](https://github.com/I-TECH-UW/OpenELIS-Global-2/blob/develop/install/createDefaultPassword.sh#L21)

Reply:
Given theres already a [default password](https://github.com/I-TECH-UW/OpenELIS-Global-2/blob/develop/src/main/resources/adminPassword.txt#L1) set , we could just leave out this script .
Otherwise ,on building this image ,it sets password as null